app/src/app/components/sidebar/charts/HorizontalBarChart.tsx

```diff
@@ -44,30 +88,49 @@ export const HorizontalBar = () => {
   return (
     <Flex gap="3" direction="column">
       <Heading as="h3" size="3">
-        Population by Zone
+        Population by District
```

Comment:
PP: looks like Moon has been using sentence case which I think is better style -> `Population by district`